src/containers/Review/assignment/AssigneeLabel.tsx

```diff
@@ -4,6 +4,7 @@ import { useLanguageProvider } from '../../../contexts/Localisation'
 
 interface AssigneeLabelProps {
   assignee: string
+  isSubmitted: boolean
```

Comment:
Sorry to be a pain -- just one comment about this:
There is the case where a consolidator requested changes from the lower level reviewer, but that reviewer is no longer available so they need to be re-assigned. By restricting to "isSubmitted" we miss that case. I was thinking a better prop would be `isCurrentLevel` (the current levelName is already in your "AssignmentSectionRow" component), so we can do re-assignments if it's gone back to that level, not just if it's submitted or not.